excel/src/main/java/tech/tablesaw/io/xlsx/XlsxReader.java

```diff
@@ -146,13 +141,16 @@ private Boolean isBlank(Cell cell) {
   }
 
   private ColumnType getColumnType(Cell cell) {
-    switch (cell.getCellType()) {
+    CellType cellType = cell.getCellType() == FORMULA ? cell.getCachedFormulaResultType() : cell.getCellType();
+    switch (cellType) {
       case STRING:
         return ColumnType.STRING;
       case NUMERIC:
         return DateUtil.isCellDateFormatted(cell) ? ColumnType.LOCAL_DATE_TIME : ColumnType.INTEGER;
       case BOOLEAN:
         return ColumnType.BOOLEAN;
+      case FORMULA:
```

Comment:
I don't think you still need this `case` anymore (same below in `appendValue`)

Reply:
yes..I missed that